app/javascript/components/rooms/CantCreateRoom.jsx

```diff
@@ -1,28 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Card, Stack } from 'react-bootstrap';
 import Button from 'react-bootstrap/Button';
 import { useTranslation } from 'react-i18next';
-import Logo from '../shared_components/Logo';
+import { useNavigate } from 'react-router-dom';
 
-// This page is shown is the current_user does NOT have CreateRoom permission
+// This page is shown if the user does NOT have the CreateRoom permission
 export default function CantCreateRoom() {
   const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  const [meetingUrl, setMeetingUrl] = useState('');
+
+  const handleChange = (event) => {
+    setMeetingUrl(event.target.value);
+  };
```

Comment:
Why the state variable? Why not just take the sibling value onClick? I think we do that in a few places already

Reply:
`handleChange` seems to be more in line with the React way to do things.

Reply:
I am also not sure what is meant by sibling value - isn't that more of a jQuery thing?

Reply:
https://github.com/bigbluebutton/greenlight/blob/v3/app/javascript/components/admin/site_settings/registration/Registration.jsx#L80

Reply:
Don't have a strong opinion so I'll leave it up to you

Reply:
I like how it's done with the useState hook